frappe/rate_limiter.py

```diff
@@ -139,6 +139,7 @@ def wrapper(*args, **kwargs):
 				frappe.throw(_("Either key or IP flag is required."))
 
 			cache_key = f"rl:{frappe.form_dict.cmd}:{identity}"
+			cache_key = frappe.cache.make_key(key)
```

Comment:
Can just do `frappe.cache.get_value()` instead?

Reply:
incr, setex etc isn't implemented afaik

Reply:
Done

Reply:
Rolled back, this isn't possible right now. We pickle everything so get_value/set_value won't work with incrby.